### PR TITLE
GCI-ajey muthiah

### PR DESCRIPTION
**Pull request by ajeyprasand**

### CODE REVIEW

File: aj.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      router := gin.Default()
+      router.GET("/hello", func(c *gin.Context) {
+c.JSON(200, gin.H{
+"message":"Hello World!",
+})
+})
+      router.GET("/greet/:name", func(c *gin.Context) {
+	name := c.Param("name")
+	c.JSON(200, gin.H{ 
+"greet-message": "Hello " + name + "!",
```
Comment (f4ww4z) [inline aj.go:15]: It should just be `"message"`.

File: aj.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package main
+
+import "github.com/gin-gonic/gin"
+
+func main() {
+      router := gin.Default()
+      router.GET("/hello", func(c *gin.Context) {
+c.JSON(200, gin.H{
```
Comment (f4ww4z) [inline aj.go:8]: Use `http.StatusOK` instead of `200`.

File: aj.go [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+func main() {
+      router := gin.Default()
+      router.GET("/hello", func(c *gin.Context) {
+c.JSON(200, gin.H{
+"message":"Hello World!",
+})
+})
+      router.GET("/greet/:name", func(c *gin.Context) {
+	name := c.Param("name")
+	c.JSON(200, gin.H{
```
Comment (f4ww4z) [inline aj.go:14]: See above comment.

File: aj.go [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+func main() {
+      router := gin.Default()
+      router.GET("/hello", func(c *gin.Context) {
+c.JSON(200, gin.H{
+"message":"Hello World!",
+})
+})
+      router.GET("/greet/:name", func(c *gin.Context) {
+	name := c.Param("name")
```
Comment (f4ww4z) [inline aj.go:13]: Make `"name"` a global constant in this file.

**Review by f4ww4z — COMMENTED** [review]
@ajeyprasand See my comments. Also please format this file, and change PR and commit message to describe what you're changing, as mentioned in the task description.

Why did you close this PR?

File: aj.go (hunk 4 above)
Comment (ajeyprasand) [reply aj.go:13]: ok

Comment (ajeyprasand) [reply aj.go:13]: should i make it an global vaariable or global constant?

**Pull request closed without merging**